### DISCUSSION TITLE
Custom chain support

### DISCUSSION THREAD

**Discussion by mqklin** [WalletConnect/walletconnect-monorepo#2778] [Developer Support] [answered]
Hello. We @ Santiment have SanR chain (our own sidechain), after upgrading to WCv2 I'm getting this error: `description:"We don't support the chainId you provided: eip155:11888"`

Question - how to support our sidechain?

**Comment by finessevanes** [comment, marked as answer]
https://docs.walletconnect.com/2.0/advanced/multichain/overview#register-chain-with-the-explorer

**Reply by mqklin** [reply]
How long will it take? 
https://github.com/WalletConnect/walletconnect-monorepo/issues/2786

**Reply by finessevanes** [reply]
We review new chains on a weekly basis, generally Fridays.

**Reply by mqklin** [reply]
@finessevanes can you do it now because of shutdown? As you made it obligatory to add all chains to use WC2, and you did not mention that this is obligatory in your mailing list, nor in your Migration 1->2 instruction.

**Reply by finessevanes** [reply]
>As you made it obligatory to add all chains to use WC2, and you did not mention that this is obligatory in your mailing list, nor in your Migration 1->2 instruction.

I'm sorry I'm not sure I understand what the concern is. We've been letting all dapps and wallets know about the upcoming migration for several months.

I pinged our team to review.

**Reply by mqklin** [reply]
> We've been letting all dapps and wallets know about the upcoming migration for several months.

I understand this. I was waiting for MetaMask iOS to release WCv2 support, as we heavily depend on it. They released it just last week.

**Reply by mqklin** [reply]
Thank you very much.
